File: src/domain/medicalInfo.ts
export interface MedicalInfo {
    id: number;
    smoker: boolean;
    last_blood_pressure_check: string;
    last_blood_pressure_measurement: string;
    has_family_history_of_diabetes: boolean;
    known_heart_condition: boolean;
    had_high_cholesterol_recently: boolean;
    overweight_amount: number;
    orthopedic_issue_location: string;
    prescribed_medication: string;
    uses_supplements: string;
    last_physical_exam_date: Date;
    does_regular_exercise: boolean;
    might_be_pregnant: boolean;
    fitness_declaration: boolean;
}

File: src/main/java/com/anatartari/tatamecontrolapi/core/model/MedicalInfo.java
package com.anatartari.tatamecontrolapi.core.model;

import lombok.*;

import java.time.LocalDate;

@Getter
@Setter
@AllArgsConstructor
@Builder
public class MedicalInfo {
    Long id;
    Boolean smoker;
    String lastBloodPressureCheck;
    String lastBloodPressureMeasurement;
    Boolean hasFamilyHistoryOfDiabetes;
    Boolean knownHeartCondition;
    Boolean hadHighCholesterolRecently;
    Double overweightAmount;
    String orthopedicIssueLocation;
    String prescribedMedication;
    String usesSupplements;
    LocalDate lastPhysicalExamDate;
    Boolean doesRegularExercise;
    Boolean mightBePregnant;
    Boolean fitnessDeclaration;
}